src/main/java/editCard/EditCardPresenter.java

```diff
@@ -1,18 +1,20 @@
-package editCard;/*
+/*
  * Author: Albert Yan
  * Date: October 31/22
  * The presenter class for edit card systems
  */
 
```

Comment:
see comment in the EditCardController class

Reply:
Done.